services/mgmt/compute/examples/list_free_linux_images.rs

```diff
@@ -24,7 +24,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let mut publishers = client
         .list_publishers(&location, &subscription_id)
-        .into_future()
+        .into_body()
```

Comment:
The convenience function that sends the request and returns the body is now `into_body`.

Reply:
Operations like `Service_GetAccountInfo` do not have response bodies. All of the retuned data is in the response headers, which are not currently accessible. The plan is make them available, probably via `Response::headers(&self)` which follows the AutoRest C# design. I'm not sure what `into_future` should map to and would prefer to add it back once we figure that out.

Reply:
When cadl replaces openapi for the specifications, it will be possible to model the responses the way we with them to be in the SDKs. I created an example for operation in https://github.com/Azure/azure-sdk-for-rust/pull/1043#issuecomment-1228933586 .